ospd/misc.py

```diff
@@ -127,6 +127,18 @@ def set_host_finished(self, scan_id, target, host):
         finished_hosts[target].extend(host)
         self.scans_table[scan_id]['finished_hosts'] = finished_hosts
 
+    def get_host_unfinished(self, scan_id):
+        """ Get the dict of targets and their finished hosts."""
```

Comment:
I guess the docstring is not correct because a list of unfinished hosts is returned here